File: CodeWars/8kyu/JavaScript/004-can-we-divide-it.js
// https://www.codewars.com/kata/5a2b703dc5e2845c0900005a
// Your task is to create functionisDivideBy (or is_divide_by) to check if an integer number is divisible by each out of two arguments.
// (-12, 2, -6)  ->  true
// (-12, 2, -5)  ->  false

function isDivideBy(number, a, b) {
    return (number % a === 0 && number % b === 0)
}

console.log(isDivideBy(-12, 2, -6), true);
console.log(isDivideBy(-12, 2, -5), false);
console.log(isDivideBy(45, 1, 6), false);
console.log(isDivideBy(45, 5, 15), true);
console.log(isDivideBy(4, 1, 4), true);
console.log(isDivideBy(15, -5, 3), true);

File: CodeWars/8kyu/Python/004-can-we-divide-it.py
# https://www.codewars.com/kata/5a2b703dc5e2845c0900005a
# create functionisDivideBy (or is_divide_by) to check if an integer number is divisible by each out of two arguments.

def is_divide_by(n, x, y):
    return n % x == 0 and n % y == 0


print(is_divide_by(8, 2, 4), True)
print(is_divide_by(12, -3, 4), True)
print(is_divide_by(8, 3, 4), False)
print(is_divide_by(48, 2, -5), False)
print(is_divide_by(-100, -25, 10), True)
print(is_divide_by(10000, 5, -3), False)
print(is_divide_by(4, 4, 2), True)
print(is_divide_by(5, 2, 3), False)
print(is_divide_by(-96, 25, 17), False)
print(is_divide_by(33, 1, 33), True)
